assets/lokomotive-kubernetes/bootkube/resources/charts/webhook-admission/values.yaml

```diff
@@ -0,0 +1,33 @@
+certificates:
+  caKey:
+  caCert:
+
+namespace: kube-system
+
+deployment:
+  replicaCount: 1
+  image:
+    repository: kinvolk/mutating-admission-controller
+    pullPolicy: IfNotPresent
+    tag: "latest"
+  securityContext:
```

Comment:
I thought it's better to have values at one place than hardcoding in chart, as it is easier to maintain

Reply:
IMO it introduces an indirection, so you need to look up two things when analyzing the chart. Since we don't plan to re-use or allow configuring those values for now, IMO it is better to keep them in-line. But I'll let others decide here.

Reply:
I am sorry, I thought you meant removing values.yaml file. securityContext can be removed from values.yaml